tortoise/ninja_tortoise.go

```diff
@@ -555,7 +552,7 @@ func (ni *NinjaTortoise) handleIncomingLayer(newlyr *types.Layer) {
 			}
 
 			tp := ni.TPattern[p]
-			ni.ForBlockInView(tp, windowStart, foo)
+			ni.db.ForBlockInView(tp, windowStart, foo)
 
 			//add corrected implicit votes
```

Comment:
why "corrected"?

Reply:
because we take into account the correction vectors we prepared we are not just adding implicit.

Reply:
I see, `updateCorrectionVectors()` only "prepare the ground" for the corrections, but here (in lines 341 to 345) we actually apply them?

Reply:
yes

Reply:
feel like commenting about it somewhere?